### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by Default4152** [EileenRice/f1-3-c2p1-colmar-academy#7]
Grade: **Meets Expectations**

Overall, this project was done well. I really like how clean your HTML and CSS is to read over. On top of that, you downloaded all of the images and put them into a resource folder which allows for faster page loading and more organization in the project. A couple things I would work on is making sure you have all of your closing tags, as well as adding alt tags to your images to provide a text descriptor for the images. From here, I would recommend adding additional pages to the project with HTML and CSS of their own.

Nice job!

**Comment by EileenRice** [comment]
Thanks for the input.  Sometimes it felt as if I hadn't retained anything.  I'll definitely try to be more specific and descriptive with the alt tags. I wanted to be concise without being too wordy.  

I appreciate you having taken the time to review my project and to do it so promptly.  Thanks again.
